web/app/omattiedot/suoritusjako/SuoritusjakoLink.jsx

```diff
@@ -112,13 +111,11 @@ export class SuoritusjakoLink extends React.Component {
             </div>
             <div className='suoritusjako-link__expiration'>
               <label htmlFor={labelId}><Text name='Linkin voimassaoloaika päättyy'/></label>
+              <div style={{display: 'inline'}} className={isDateUpdatePending ? 'ajax-indicator-right' : ''}>
               <DateInput
                 value={parseISODate(expirationDate)}
                 valueCallback={date => this.dateChangeBus.push(date)}
-                validityCallback={(isValid, stringInput)=> {
-                  !isValid && this.dateChangeBus.push(null)
```

Comment:
Epävalidin syötteen kohdalla muutosten "nollaamista" (`this.dateChangeBus.push(null)`) ei tule poistaa: ilman tätä käyttöliittymä ja todellinen tila päätyvät epäsynkkaan tilanteessa, jossa syöttökentän osalta
1) aloitetaan epävalidista syötteestä
2) syötetään validi syöte
3) syötetään heti perään epävalidi syöte (nyt edellistä validia syötettä ei "nollata")
=> edelliset (vaiheen 2) muutoksen on tallennettu ja näytetään "Muutoksen tallennettu", vaikka syötekentässä on tällä hetkellä epävalidi syöte